### ISSUE TITLE
ERROR: UNEXPECTED VALUE 0 FOR RADIO TYPE

### ISSUE THREAD

**Issue by laendi** [kersing/multitech-installer#13]
Hi I have a Multitech AP with mlinux Version 3.1.13

I tried to connect to TTN but I doesnt work. Here's the error.

13:40:57  INFO: [main] Starting the concentrator
lgw_start:807: ERROR: UNEXPECTED VALUE 0 FOR RADIO TYPE
lgw_start:833: Note: calibration started (time: 2300 ms)
lgw_start:854: Note: calibration finished (status = 135)
ERROR: wrong configuration, rf_rx_freq[0] is not set
INFO: FPGA supported features: [TX filter]  [Spectral Scan] 
13:41:00  ERROR: [main] failed to start the concentrator

Could you help me?

Kind regards

**Comment by kersing** [comment]
How did you install the software? The errors indicates your global_conf.json or local_conf.json do not contain the required information.

**Comment by laendi** [comment]
Hi

In installed it how it is described on the TTN documentation. But with wget ... and then sh installer.sh.

There is no usb port as described in the documentation for multitech conduit. 

Is there another documentation ? 

Kind regards

Stefan


—
Stefan Landolt
product things

thingdust AG
Moosstrasse 7
6003 Luzern

+41 79 718 19 92
stefan@thingdust.com
thingdust.com

**Comment by terrillmoore** [comment]
Stefan, we are successfully installing on the Conduit AP, but we are using a different procedure. We are using Ansible to manage the Conduits, which is a fully automated procedure. This means that our docs are not completely suitable to people who are doing a one-off installation.

The Ansible setup we have is at https://github.com/IthacaThings/ttn-multitech-cm -- based on current experimentation, although it's working for @jchonig and me, it probably not quite ready for prime time.

For the Conduit AP, you have to use a pure Ethernet-based install. There is some more information at https://github.com/terrillmoore/conduit-mfg and https://github.com/terrillmoore/conduit-mfg/blob/master/CONDUIT-AP.md -- but again, that's biased for the Ansible-based install. 

At The Things Network Ithaca/NYC, we have done our own mLinux distro, to facilitate remote update -- the stock mLinux loses key settings during an upgrade (at least the ones critical for us for remote management). This distro includes the Kersing forwarder, and it works. (It's also a lot smaller and simpler than the stock MultiTech distribution).

So to summarize, you may find helpful info at:

- https://github.com/IthacaThings/ttn-multitech-cm
- https://github.com/terrillmoore/conduit-mfg
- https://github.com/terrillmoore/conduit-mfg/blob/master/CONDUIT-AP.md

Feel free to go to https://thethings.nyc and follow the instructions for joining our slack workgroup to get support.

--Terry
terry@thethings.nyc

**Comment by kersing** [comment]
@terrillmoore Terry, I appreciate help, however I don’t think pointing people to alternative instructions based on anonther installation method will help much.

@laendi can you check what files are in /var/config/lora and attach global_conf.json if present?

**Comment by laendi** [comment]
Hi

In

root@mtcap:~# ls /var/config/lora
global_conf_src       local_conf.json       ttn_global_conf.json



file content:

{"SX1301_conf":{"lorawan_public":true,"clksrc":1,"clksrc_desc":"radio_1 provides clock to concentrator for most devices except MultiTech. For MultiTech set to 0.","antenna_gain":$


Kind regards

Stefan


On 29 Dec 2017, at 23:18, Jac Kersing <notifications@github.com<mailto:notifications@github.com>> wrote:


@terrillmoore<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fterrillmoore&data=02%7C01%7C%7Cc7f5224cb81a434910ea08d54f0a1327%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636501827038070438&sdata=WCA0x7H24XYGKRmkVuz4u5AgFfBnTzzU6xhuRVHGvK4%3D&reserved=0> Terry, I appreciate help, however I don’t think pointing people to alternative instructions based on anonther installation method will help much.

@laendi<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Flaendi&data=02%7C01%7C%7Cc7f5224cb81a434910ea08d54f0a1327%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636501827038070438&sdata=ZITgdwHkTtIeExRQATlNgLsX7dm9Lllc09594wQ%2BwIo%3D&reserved=0> can you check what files are in /var/config/lora and attach global_conf.json if present?

**Comment by kersing** [comment]
Hi Stefan,

Please run ‘node -v’ to check the version of node-js on the box.

**Comment by laendi** [comment]
There was no node-js installed. Now I installed it and it’s the version v0.10.44

Kind regards

Stefan

On 30 Dec 2017, at 20:27, Jac Kersing <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi Stefan,

Please run ‘node -v’ to check the version of node-js on the box.

**Comment by laendi** [comment]
Now it worked. Thank you
